magefiles/test.go

```diff
@@ -89,6 +90,11 @@ func testImpl(impl string) error {
 		return errors.New("unknown `gokv.Store` implementation")
 	}
 
+	// TODO: until docker images for windows appear, skip those test for windows
```

Comment:
It's unlikely that there will be Windows specific images, so I was hoping we could either
- Switch the Docker daemon to Linux mode (like [this](https://stackoverflow.com/a/57081502)), but apparently the GitHub Action Windows runners don't support it ([source](https://github.com/orgs/community/discussions/25491))
- Or run Docker in [WSL 2](https://docs.docker.com/desktop/wsl/), but apparently the GitHub Action Windows runners only support WSL 1 ([source](https://github.com/actions/runner-images/issues/1143))

So we're out of luck. Let's see if the situation improves in the future. For now it's already a win to test the other gokv stores that don't rely on services running in Docker :+1:

Reply:
I found one for mongo
https://hub.docker.com/_/mongo/tags?page=1&name=windows
but yeah, was exploring both options for above points you made as well